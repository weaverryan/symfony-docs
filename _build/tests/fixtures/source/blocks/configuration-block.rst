
.. configuration-block::

    .. code-block:: yaml

        # app/config/services.yml

    .. code-block:: php

        // config/routes.php
