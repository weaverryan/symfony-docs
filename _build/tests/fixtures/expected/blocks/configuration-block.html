<!DOCTYPE html>
<html>
<head>
    <meta charset="utf-8" />
</head>
<body>
<div class="configuration-block"><ul class="simple"><li><em>YAML</em><div class="literal-block notranslate">
    <div class="highlight-yaml">
        <table class="highlighttable">
            <tr>
                <td class="linenos">
                    <div class="linenodiv">
                        <pre> 1</pre>
                    </div>
                </td>
                <td class="code">
                    <div class="highlight">
                        <pre># app/config/services.yml</pre>
                    </div>
                </td>
            </tr>
        </table>
    </div>
</div></li><li><em>PHP</em><div class="literal-block notranslate">
                <div class="highlight-php">
                    <table class="highlighttable">
                        <tr>
                            <td class="linenos">
                                <div class="linenodiv">
                                    <pre> 1</pre>
                                </div>
                            </td>
                            <td class="code">
                                <div class="highlight">
                                    <pre>// config/routes.php</pre>
                                </div>
                            </td>
                        </tr>
                    </table>
                </div>
            </div>
        </li>
    </ul>
</div>
</body>
</html>
